sql/core/src/test/scala/org/apache/spark/sql/DateFunctionsSuite.scala

```diff
@@ -828,4 +828,17 @@ class DateFunctionsSuite extends QueryTest with SharedSparkSession {
       TimeZone.setDefault(defaultTz)
     }
   }
+
+  test("truncate timestamps before the epoch to seconds and minutes") {
```

Comment:
Can you add the jira number in the prefix?

Reply:
Added JIRA id to the test title.